lib/services/auth-service.js

```diff
@@ -67,6 +58,24 @@ function authServiceFactory(
             secretOrKey : self.secretOrKey,
             ignoreExpiration : self.ignoreTokenExpiration
         };
+
+        // used to serialize the user for the session
```

Comment:
@brianparry Rather than using passport.session() middleware, I'm calling the logIn function directly and passing it the username:https://github.com/RackHD/on-http/pull/178/files#diff-a5da55608b028f454f167e0a83bbfabeR239.  logIn will call serializeUser with the parameter it is passed (the passport middleware normally calls logIn with the session object) .  In the case of my code, serializeUser is really only used to return success, as it needs to return successfully for logIn to succeed.

As for deserializeUser, that isn't used at all and can be deleted.  I'll remove deserializeUser and update the comments around serializeUser.